tests/foreman/ui/test_subscription.py

```diff
@@ -554,3 +556,79 @@ def test_positive_prepare_for_sca_only_subscription(target_sat, function_entitle
             'This organization is not using Simple Content Access. Entitlement-based subscription '
             'management is deprecated and will be removed in Satellite 6.16.' in sca_alert
         )
+
+
+@pytest.mark.parametrize('setting_update', ['expire_soon_days'], indirect=True)
+def test_positive_check_manifest_validity_notification(
+    target_sat, setting_update, function_org, function_sca_manifest, default_org
+):
+    """Check notification when manifest is going to expire.
+
+    :id: 29ab0e80-25eb-44f0-9294-3a26922c33c5
+
+    :setup:
+        1. create new organization
+
+    :steps:
+        1. Upload expired manifest in newly created org
+        2. go to Content > Subscriptions page, click on 'Manage Manifest' button.
+        3. Search for message string 'Manifest expired', or 'Your manifest expired'
+        4. delete expired manifest from this org (cleanup part)
+        5. Upload non-expired manifest
+        6. go to Content > Subscription page, click on 'Manage Manifest' button.
+        7. Search for message string 'Manifest expiring soon', or 'Your manifest will expire'
+        8. delete expired manifest from this org (cleanup part)
+
+    :expectedresults:
+        1. 'Manifest expired', 'Manifest expiring soon' messages appear on Manage Manifest modal box
+
+    :BZ: 2075163
+
+    :customerscenario: true
+    """
+    remote_path = f'/tmp/{EXPIRED_MANIFEST}'
+    target_sat.put(DataFile.EXPIRED_MANIFEST_FILE, remote_path)
+    # upload expired manifest
+    target_sat.cli.Subscription.upload({'organization-id': function_org.id, 'file': remote_path})
+
+    with target_sat.ui_session() as session:
+        # Message - Manifest expired
+        session.organization.select(function_org.name)
+        # read expire manifest message
+        expired_manifest = session.subscription.read_subscription_manifest_header_message_and_date()
+        assert 'Manifest expired' in expired_manifest['header'], 'Manifest expire alert not found'
+        assert 'Your manifest expired' in expired_manifest['message']
+        assert 'import a new manifest' in expired_manifest['message']
```

Comment:
I think the expiration date is a constant for the given manifest. What do you think about adding it to constants and hardening the test with check for the correct date?
```suggestion
        assert (
            f'Your manifest expired on {constants.EXPIRED_MANIFEST_DATE}. To continue '
            'using Red Hat content, import a new manifest.'
        ) in expired_manifest['message']
```